### DISCUSSION TITLE
Choosing the dimension of Krylov space for a Spin system of Length N in Lanczos Method

### DISCUSSION THREAD

**Discussion by hcut** [QuSpin/QuSpin#560] [Q&A]
Hello QuSpin,

For a given length N of spin chain ( Let's take spin 1/2, then dimension of Hilbert space $2^N$ ), How dimension of Krylov space (m) is chosen ?

Thanks.

**Comment by weinbe58** [comment]
It depends on the spectrum of your Hamiltonian; I would suggest checking the convergence with a few values of m. To do this you can take the matrices given by the lanczos code and reconstruct the original krylov Hamiltonian and then disgonalizing that matrix truncating the number of rows to get multiple values of m
